packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -265,20 +245,23 @@ export default {
     }
   },
   methods: {
-    ...mapActions('Files', ['updateFileProgress', 'loadIndicators']),
-    ...mapActions(['showMessage', 'createModal', 'setModalInputErrorMessage']),
+    ...mapActions('Files', ['loadPreview', 'loadIndicators']),
+    ...mapActions(['openFile', 'showMessage', 'createModal', 'setModalInputErrorMessage']),
     ...mapMutations('Files', ['UPSERT_RESOURCE']),
     ...mapMutations(['SET_QUOTA']),
 
-    async onFileSuccess(event, file) {
+    async onFileSuccess(file: UppyResource) {
```

Comment:
Could you provide some steps to reproduce this? I tried several times, but I couldn't get this error.

Reply:
Hmm could be. I noticed another strange behavior though: When you upload `file.txt` to a public page, reload, then upload the same file again and overwrite it, it will appear twice in the file list. It happens on master as well